pushapkscript/task.py

```diff
@@ -8,15 +8,49 @@
 
 
 def extract_android_product_from_scopes(context):
-    prefix = context.config['taskcluster_scope_prefix']
+    prefixes = _get_scope_prefixes(context)
+    scopes = _extract_scopes_from_unique_prefix(
```

Comment:
I don't quite understand why this is a separate function - since we still want a single scope, why can't we keep the old behaviour of using `condition` in `get_single_item_from_sequence(...)`?

Reply:
Yeah, you're right. I don't remember why I made these functions this way, in the first place. However, they are too convoluted for this use case. Thank you pointing this out!